uniflow/op/model/cv/model_server.py

```diff
@@ -262,3 +268,214 @@ def __call__(self, data: List[str]) -> List[str]:
+
+        Returns:
+            List[str]: Preprocessed data.
+        """
+        return data
+
+    def _postprocess(self, data: List[str]) -> List[str]:
+        """Postprocess data.
+
+        Args:
+            data (List[str]): Data to postprocess.
+
+        Returns:
+            List[str]: Postprocessed data.
+        """
+        return data
+
+    def __call__(self, data: List[str]) -> List[str]:
+        """Run model.
+
+        Args:
+            data (List[str]): Data to run.
+
+        Returns:
+            List[str]: Output data.
+        """
+
+        import os
+
+        import nest_asyncio
+
+        nest_asyncio.apply()
+
+        from llama_parse import LlamaParse
+
+        parser = LlamaParse(
+            api_key=os.getenv("LLAMA_CLOUD_API_KEY"),
+            result_type=self._model_config.result_type,  # "markdown" and "text" are available
+            num_workers=self._model_config.num_wokers,  # if multiple files passed, split in `num_workers` API calls
+            verbose=True,
+            language=self._model_config.language,  # Optionally you can define a language, default=en
+        )
+
+        out = []
+
+        if self._model_config.sync:
+            for pdf in data:
+                result = parser.load_data(pdf)
+                out.append(result[0].text)
+        else:
+            # TODO: implement async
+            print("running llamaparse async...")
+
+        return out
+
+
+class OpenParserModelServer(AbsModelServer):
+    """Open-parser Model Server"""
+
+    def __init__(
+        self, prompt_template: PromptTemplate, model_config: Dict[str, Any]
+    ) -> None:
+        super().__init__(prompt_template, model_config)
+        self._model_config = OpenParserModelConfig(**self._model_config)
+
+        try:
+            from open_parser import (  # pylint: disable=import-outside-toplevel
+                OpenParser,
+            )
+
+        except ModuleNotFoundError as exc:
+            raise ModuleNotFoundError(
+                "Please install open_parser to use OpenParserModelServer. You can use `pip install open_parser` to install it."
+            ) from exc
+
+    def _preprocess(self, data: str) -> List[str]:
+        """Preprocess data.
+
+        Args:
+            data (List[str]): Data to preprocess.
+
+        Returns:
+            List[str]: Preprocessed data.
+        """
+        return data
+
+    def _postprocess(self, data: List[str]) -> List[str]:
+        """Postprocess data.
+
+        Args:
+            data (List[str]): Data to postprocess.
+
+        Returns:
+            List[str]: Postprocessed data.
+        """
+        return data
+
+    def __call__(self, data: List[str]) -> List[str]:
+        """Run model.
+
+        Args:
+            data (List[str]): Data to run.
+
+        Returns:
+            List[str]: Output data.
+        """
+        import os
+
+        from open_parser import OpenParser
+
+        cambioml_api_key = os.getenv("CAMBIO_API_KEY")
```

Comment:
same as above. consider to pass through config because people might name it as other ENV name.